docs/advanced/document-store/benchmark.md

```diff
@@ -10,18 +9,19 @@
 :scale: 0 %
 ```
 
-We create a DocumentArray with one million Documents using randomly generated data from a uniform distribution and benchmark all supported document stores.
+We create a DocumentArray with one million Documents based on [sift1m](http://corpus-texmex.irisa.fr/), a dataset containing 1 million objects of 128d and using l2 distance metrics, and benchmark the following document stores.
+
 This includes classic database and vector database, all under the same DocumentArray API:
 
-| Name                                | Usage                                    | Version           |
-|-------------------------------------|------------------------------------------|-------------------|
-| In-memory DocumentArray             | `DocumentArray()`                        | DocArray `0.16.0` |
-| [`Sqlite`](sqlite.md)               | `DocumentArray(storage='sqlite')`        | `2.6.0`           |
-| [`Weaviate`](weaviate.md)           | `DocumentArray(storage='weaviate')`      | `3.3.0`           |
-| [`Qdrant`](qdrant.md)               | `DocumentArray(storage='qdrant')`        | `0.7.3`           |
-| [`Annlite`](annlite.md)             | `DocumentArray(storage='anlite')`        | `0.3.2`           |
-| [`ElasticSearch`](elasticsearch.md) | `DocumentArray(storage='elasticsearch')` | `8.2.0`           |
-| [`Redis`](redis.md)                 | `DocumentArray(storage='redis')`         | `4.3.0`           |
+| Name                                | Usage                                    | Client Version    | Database Version |
+|-------------------------------------|------------------------------------------|-------------------|------------------|
+| In-memory DocumentArray             | `DocumentArray()`                        | DocArray `0.18.2` | N/A              |
+| [`Sqlite`](sqlite.md)               | `DocumentArray(storage='sqlite')`        | `2.6.0`           | N/A              |
+| [`Weaviate`](weaviate.md)           | `DocumentArray(storage='weaviate')`      | `3.3.3`           | `1.13.2`         |
+| [`Qdrant`](qdrant.md)               | `DocumentArray(storage='qdrant')`        | `0.8.0`           | `0.8.0`          |
+| [`Annlite`](annlite.md)             | `DocumentArray(storage='anlite')`        | `0.3.13`          | N/A              |
+| [`ElasticSearch`](elasticsearch.md) | `DocumentArray(storage='elasticsearch')` | `8.4.3`           | `8.2.0`          |
+| [`Redis`](redis.md)                 | `DocumentArray(storage='redis')`         | `4.3.4`           | `2.6.0`          |
 
 We focus on the following tasks:
 
```

Comment:
and for the filter operation, let's say it uses backend side filtering when possible

Reply:
in the text below, instead of saying:

> We use similar underlying search algorithms but different implementations**: In this benchmark we focus on setting only parameters `ef`, `ef_construct` and `max_connections` from HNSW

let's say

> We use similar underlying search algorithms but different implementations**: In this benchmark we try a set of parameters `ef`, `ef_construct` and `max_connections` from HNSW applied equally on all backends.